datacenter/dtr/2.2/guides/admin/configure/garbage-collection.md

```diff
@@ -21,7 +21,7 @@ UTC time (unless the system time is mounted), therefore remember that the cron
 schedule will run based off of UTC time when configuring.
 
 GC puts DTR into read only mode; pulls succeed while pushes fail.  Pushing an
-image while GC runs may lead to undefined behaviour and data loss, therefore
+image while GC runs may lead to undefined behavior and data loss, therefore
```

Comment:
US vs CW English. Do we specify US in the style guide? Same comment for "labelled", "cancelled" below.